src/nodes/Token.hack

```diff
@@ -144,7 +144,8 @@ abstract class Token extends Node {
     $trailing_list = __Private\fold_map(
       /* HH_IGNORE_ERROR[4110] */ $json['trailing'],
       ($j, $p) ==> Trivia::fromJSON($j, $file, $p, $source, 'Node'),
-      ($j, $p) ==> $j['width'] + $p,
+      /* HH_FIXME[4297] use like-types when available*/
+      ($j, $p) ==> $j as shape('width' => int, ...) + $p,
```

Comment:
is this fixme still needed?

Reply:
:slightly_smiling_face: no, I should have deleted it.